README.md

```diff
@@ -15,6 +15,11 @@
 
 [Live Demo Link](https://bruk19.github.io/Capstone-project/)
```

Comment:
Your live link is functional! Well done using GitHub Pages to deploy your website 💣